modules/cloudpinyin/cloudpinyin.cpp

```diff
@@ -118,8 +118,11 @@ CloudPinyin::~CloudPinyin() { dispatcher_.detach(); }
 
 void CloudPinyin::reloadConfig() {
     readAsIni(config_, "conf/cloudpinyin.conf");
+    populateConfig();
 }
 
+void CloudPinyin::populateConfig() {}
```

Comment:
?? what's the point to have a empty function

Reply:
To keep consistency, so I want to call populateConfig in all addons's reloadConfig, although some addons does not need populateConfig. If you think it is unnecessary, I will be remove it.